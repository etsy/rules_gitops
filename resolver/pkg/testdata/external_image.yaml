apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: etcd
  name: etcd-deployment
spec:
  replicas: 3
  selector:
    matchLabels:
      app: etcd
  template:
    metadata:
      labels:
        app: etcd
    spec:
      containers:
        - image: "@etcd"
          name: etcd
          ports:
            - containerPort: 80
