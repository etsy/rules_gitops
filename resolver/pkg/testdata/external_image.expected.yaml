apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: etcd
  name: etcd-deployment
spec:
  replicas: 3
  selector:
    matchLabels:
      app: etcd
  template:
    metadata:
      labels:
        app: etcd
    spec:
      containers:
      - image: docker.io/library/etcd:tag
        name: etcd
        ports:
        - containerPort: 80